pcap_bw.py

```diff
@@ -96,6 +102,7 @@ def inet_to_str(inet):
             totbw += pkt.len
             src = inet_to_str(pkt.src)
             dst = inet_to_str(pkt.dst)
-            hostbw[src][dst] += pkt.len
+            if src in hostbw and dst in hostbw[src]:
+                hostbw[src][dst] += pkt.len
```

Comment:
This is not necessary now, and the membership tests will just slow things down. 
By construction `hostbw` is a dense square matrix, with rows and columns indexed by elements of the list `HOSTS`.
We could generalise to only considering the traffic matrix between a subset of hosts I guess, but we don't need to yet. (And I'd want to benchmark doing that via a try/except block vs doing it this way in that case.)
Could perhaps add a note in the help text to the effect that `HOSTS` is currently expected to be complete for the trace. (Or write a tool to do a separate pass to extract it for later feeding in.)